cmake/quickjs.cmake

```diff
@@ -28,8 +28,10 @@ if("sgx" IN_LIST COMPILE_TARGETS)
     quickjs.enclave STATIC ${QUICKJS_SRC} ${CCF_DIR}/3rdparty/stub/time.c
   )
   target_compile_options(
-    quickjs.enclave PUBLIC -nostdinc -DCONFIG_VERSION="${QUICKJS_VERSION}"
-                           -DEMSCRIPTEN -DCONFIG_STACK_CHECK
+    quickjs.enclave
+    PUBLIC -nostdinc -DCONFIG_VERSION="${QUICKJS_VERSION}" -DEMSCRIPTEN
+           -DCONFIG_STACK_CHECK
+    PRIVATE $<$<CONFIG:Debug>:-DDUMP_LEAKS>
```

Comment:
Just to check, this is saying "when building this target in Debug, set DUMP_LEAKS", right?

Reply:
Yeah